index.js

```diff
@@ -255,6 +255,41 @@ exports.Transform = class Transform extends stream.Transform {
 
 exports.PassThrough = class PassThrough extends exports.Transform {}
 
+exports.finished = function finished (_stream, opts, cb) {
+  if (typeof opts === 'function') {
+    cb = opts
+    opts = {}
+  }
+
+  if (!opts) opts = {}
+
+  const {
+    cleanup = false
+  } = opts
+
+  const done = () => {
+    cb(stream.getStreamError(_stream, { all: true }))
```

Comment:
the `{ all: true }` is related with: https://github.com/mafintosh/streamx/pull/91